dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -66,6 +66,7 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "C&ouper", IDM_CUT
         MENUITEM "&Copier", IDM_COPY
+        MENUITEM "Paste", IDM_INSERT
```

Comment:
```
         MENUITEM "&Couper", IDM_CUT
         MENUITEM "Co&pier", IDM_COPY
        MENUITEM "C&oller", IDM_INSERT
```
cf https://github.com/reactos/reactos/blob/c2c66aff7dacc62d125f2cd61d1167e9a2aa3fd6/base/applications/wordpad/lang/fr-FR.rc#L42

Reply:
Thanks. :)
I copied all the missing strings from en-US since I didn't want to go to each individual language and look for other sources... (That's our normal practice w.r.t translations -- insert english text and let translators fix)